test/index.js

```diff
@@ -26,8 +26,11 @@ describe('default resolution', () => {
       subscription: new GraphQLObjectType({
         name: 'Subscription',
         fields: {
-          foo: subscriptionWithClientId({
+          foo: subscription({
             name: 'FooSubscription',
+            inputFields: {
+              dummy: { type: GraphQLString },
```

Comment:
If Relay doesn't check that when it's printing subscriptions/operations maybe we can change that.